### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ostUp**
[DEMO LINK](https://ostUp.github.io/react_todo-app-loading-todos/)

### CODE REVIEW

**Review by Roman-Artiukh — APPROVED** [review]
nice

File: src/App.tsx [hunk 1]
```diff
@@ -1,223 +1,55 @@
...
-          <button
-            data-cy="ClearCompletedButton"
-            type="button"
-            className="todoapp__clear-completed"
-          >
-            Clear completed
-          </button>
-        </footer>
+        {todos && (
+          <Footer
```
Comment (AndriiRut) [inline src/App.tsx:44]:
```suggestion
          <Filters
```
The footer is for the page only, not for the app. In the checklist, you have a hint about this naming.

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  const newTodoField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (newTodoField.current) {
+      newTodoField.current.focus();
+    }
+  }, []);
+
+  return (
+    <header className="todoapp__header">
```
Comment (AndriiRut) [inline src/components/Header/Header.tsx:21]: we have header on top of the page!
we can have `head` of the app or `NewTodo` component (Checklist)

**Review by AndriiRut — CHANGES_REQUESTED** [review]
on the first load of the page, I can see an error for 3 sec.
image

and some more in the comments

**Pull request closed without merging**